packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/base/common_services/users_service.dart

```diff
@@ -0,0 +1,46 @@
+import '../models/user_model.dart';
+import '../models/user_with_auth_token_model.dart';
+import '../repositories/url_launcher_repository.dart';
+import '../repositories/users_repository.dart';
+
+const mockEmailDeepLinkError =
+    '{{project_name}}://{{project_name}}/onboarding/email-confirmed/00000000';
+const mockEmailDeepLinkSuccess =
+    '{{project_name}}://{{project_name}}/onboarding/email-confirmed/11111111';
+
+class UsersService {
```

Comment:
It would be great if we had the methods and fields documented.

Reply:
Done